pycqed/tests/test_spectroscopy_analysis.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+import pycqed as pq
+import os
+from pycqed.analysis import measurement_analysis as ma
+
+
+class Test_Homodyne_analysis(unittest.TestCase):
```

Comment:
happy you added some tests for this 👍 . 

Are you familiar with `pycqed/tests/test_heterodyne_analysis` (bad name I know)? It does basically the same. 

I think more tests is better in this case (especially since you tests specific models). Could you merge these two files?